gcloud/bigquery/job.py

```diff
@@ -919,3 +931,278 @@ def _scrub_local_properties(self, cleaned):
                 assert dest_remote['projectId'] == self.project
                 dataset = self._client.dataset(dest_remote['datasetId'])
                 self.destination_table = dataset.table(dest_remote['tableId'])
+
+
+class _SyncQueryConfiguration(object):
+    """User-settable configuration options for synchronous query jobs."""
+    # None -> use server default.
```

Comment:
I'm not 100% sure, but I interpret this to mean that a RHS value of `None` means let the server default win, yes?

Also, why not put this in the docstring instead of in a comment.